client/extensions/woocommerce/state/sites/settings/email/test/helpers.js

```diff
@@ -0,0 +1,72 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { checkEmails } from 'woocommerce/app/settings/email/email-settings/components/helpers';
+
+describe( 'helpers', () => {
+	describe( '#checkEmails', () => {
+		test( 'should return error false on empty string.', () => {
+			expect( checkEmails( '' ) ).to.eql( { error: false } );
```

Comment:
do we need a test for spaces? `checkEmails( ' ' )`

Reply:
No we do not. In that case woocommerce uses default value.